source/main.c

```diff
@@ -56,6 +58,84 @@ void dol_alloc(int size)
     }
 }
 
+void load_parse_cli()
+{
+    int path_length = strlen(path);
+    path[path_length - 3] = 'c';
+    path[path_length - 2] = 'l';
+    path[path_length - 1] = 'i';
+
+    kprintf("Reading %s\n", path);
+    FIL file;
+    FRESULT result = f_open(&file, path, FA_READ);
+    if (result != FR_OK)
+    {
+        if (result == FR_NO_FILE)
+        {
+            kprintf("CLI file not found\n");
+        }
+        else
+        {
+            kprintf("Failed to open CLI file\n");
+        }
+        return;
+    }
+
+    size_t size = f_size(&file);
+    kprintf("CLI size is %iB\n", size);
+
+    if (size <= 0)
+    {
+        kprintf("Empty CLI\n");
+        return;
+    }
+
+    char *cli = (char *) memalign(32, size + 1);
```

Comment:
Is it necessary to memalign here? We memalign `dolargs.commandLine` again later.

Reply:
There is no memory alignment requirement with this stub.

Reply:
We are still aligning in `dol_alloc`, is that not needed either?

Reply:
Removed unneeded memory alignment: 5a27ce7